pkg/cable/libreswan/libreswan.go

```diff
@@ -283,6 +293,16 @@ func (i *libreswan) ConnectToEndpoint(endpoint types.SubmarinerEndpoint, useIP s
 	leftSubnets := extractSubnets(i.localEndpoint.Spec)
 	rightSubnets := extractSubnets(endpoint.Spec)
 
+	rightNATTPort, err := cable.GetBackendPort(endpoint, subv1.UDPPortBackendConfig, i.ipSecNATTPort)
+	if err != nil {
+		return "", errors.Wrapf(err, "error parsing endpoint %q IKE port", endpoint.Spec.CableName)
+	}
```

Comment:
Since `GetBackendPort` returns the specified default value, perhaps we should just log the error and not return it and use the default.  If we return the error, the engine will propagate to the tunnel controller which will queue for retry however, in this case, a retry won't help.